i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/users/README.md

```diff
@@ -164,4 +164,45 @@ _password_encrypted_ 和 _password_encryption_method_ 的示例，明文密码
 
 ### custom_data
 
+你可以使用 _custom_data_ 来做以下这些事情：
+
+- 记录用户是否执行过特定的操作，例如用户是否已经看过欢迎页面。
+- 在用户资料中保存关于特定应用的数据，例如用户在某个应用下的首选语言和外观。
+
```

Comment:
As you mentioned not to put sensitive data in custom_data, I would expect you should let users know that they can put arbitrary data in custom_data. But this is with HIGH RISK (as you mentioned in the "caution" part).

Reply:
Updated. PTAL